ooniapi/services/ooniprobe/src/ooniprobe/main.py

```diff
@@ -66,8 +95,18 @@ class HealthStatus(BaseModel):
 async def health(
     settings=Depends(get_settings),
     db=Depends(get_postgresql_session),
+    clickhouse=Depends(get_clickhouse_session),
 ):
     errors = []
+    try:
+        query = """SELECT *
+        FROM fastpath FINAL
```

Comment:
This query is incredibly heavy to run. I would not suggest running this as part of the health check as it will kill the database.

I would avoid placing any `FINAL` query in here and only place one that will return a subset of all the rows.

Maybe a reasonable query could be:
```
SELECT
COUNT()
FROM fastpath
WHERE measurement_start_time < NOW() AND measurement_start_time > NOW() - INTERVAL 3 HOUR
```

And check that you get some positive number in here.

Reply:
I agree with this change. I made the original query based on this healthcheck from `oonimeasurements`: https://github.com/ooni/backend/blob/5dec72a5c4d47cebdd0b5969758efed5bd8da9b8/ooniapi/services/oonimeasurements/src/oonimeasurements/main.py#L81-L91

Should I change that one as well?

Reply:
Yes we should change that one too. I must have missed that in previous reviews